src/frontend/src/lib/components/core/Footer.svelte

```diff
@@ -34,20 +37,24 @@
 	</div>
 
 	<div
-		class="item flex flex-row items-center justify-end gap-2 text-sm lg:max-w-48 xl:max-w-none"
+		class="item flex flex-row items-center justify-end gap-2 text-sm transition-all duration-200 ease-in-out lg:max-w-48 xl:max-w-none"
 		class:sm:max-w-none={$authNotSignedIn}
+		class:sm:invisible={$authSignedIn}
+		class:1.5md:visible={$authSignedIn}
+		class:translate-x-full={$authSignedIn}
+		class:1.5md:translate-x-0={$authSignedIn}
 	>
-		<IconDfinity />
-		<span>
-			{replaceOisyPlaceholders($i18n.footer.text.developed_with)}
-			<ExternalLink
-				href="https://dfinity.org"
-				ariaLabel={replaceOisyPlaceholders($i18n.footer.alt.dfinity)}
-				color="blue"
-				iconVisible={false}
-			>
-				{replaceOisyPlaceholders($i18n.footer.text.dfinity)}
-			</ExternalLink>
-		</span>
+		<ExternalLink
+			href="https://dfinity.org"
```

Comment:
Wait what, I missed that in a previous review. It's a link to DFINITY, not the Internet Computer?!!??!? That's correct and expected?

Reply:
I am pretty sure... I will double confirm, but the string says `Developed with... by DFINITY`, so I would imagine that is where it should point. But good question, I will ask

Reply:
Please do. Thank in advance.

Reply:
Just as follow up: I checked with both PM and Design team and that is indeed on purpose all DFINITY related.

Reply:
Thanks for double checking.